src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -165,18 +165,29 @@ public String getEncoding() {
 		return this.pkColumnNames;
 	}
 
-	public void setPK(List<String> pkColumnNames) throws SchemaSyncError {
+	public Object getPKString() {
+		if ( this.pkColumnNames != null )
+			return StringUtils.join(pkColumnNames.iterator(), ",");
```

Comment:
Do we need to qualify the column-names with quotes? Is it possible to have a column called "a,b" ?